app/helpers/groups_helper.rb

```diff
@@ -0,0 +1,13 @@
+module GroupsHelper
+  def active_users
+    @viewmodel.group.memberships.where(end: nil).includes(:user).each do |membership|
```

Comment:
Using instance variables in helpers are a clear way to create brittle code. Why not have an explicit dependency?

```ruby
def active_users(group)
# same as above
end
```